kong/plugins/datadog/schema.lua

```diff
@@ -79,6 +79,9 @@ return {
           { service_name_tag = { type = "string", default = "name" }, },
           { status_tag = { type = "string", default = "status" }, },
           { consumer_tag = { type = "string", default = "consumer" }, },
+          { retry_count = { type = "integer", default = 10 }, },
```

Comment:
I was about to ask where the default values come from, but I found them here: https://github.com/Kong/kong/blob/7397d6ffdbed91ccf2d6077dc80d992d162bb684/kong/plugins/http-log/schema.lua#L17-L19

: )